templates/community.jade

```diff
@@ -5,22 +5,26 @@ block title
 
 block content
   .content
-    section.single-column
+    section.bottom-hr.single-column
       h2 Community
       p Visit us on IRC in <a href='http://webchat.freenode.net/?channels=hapi'>#hapi</a> on <a href='http://freenode.net'>freenode</a>.
       p We also have a <a href="https://gitter.im/hapijs/hapi" target="_blank">Gitter instance</a> as well (GitHub account required).
       p hapi's stability and reliability is empowering these companies today. If you're using hapi too we'd love to <a href='https://github.com/hapijs/hapi/issues/1613'>hear from you</a>.
 
-      each company, title in community.all
-        section.company.cf
-          if (company.logo)
-            a(href='#{company.url}', class='community-company-logo')
-              img(src='/public/img/#{company.logo}', width=(company.width ? '#{company.width}' : undefined), height=(company.height ? '#{company.height}' : undefined), alt='#{title}')
-            blockquote
-              p= company.quote
-              small &ndash; #{company.person} at #{title}
-          else
-            blockquote.full-width
-              p= company.quote
-              small &ndash; #{company.person} at 
-                a(href='#{company.url}')= title
+      - var titles = Object.keys(community.all)
```

Comment:
`community.all` is an object, with property names as company titles. Iterate over the titles (stored in a variable to avoid repeated (`Object.keys(community.all).length` lookups) to gain access to the `index`, then ensure the `.bottom-rule` class isn’t applied to the last `section.company`.